### DISCUSSION TITLE
Support request: ZSH aliase suggestions is not working anymore

### DISCUSSION THREAD

**Discussion by nlvedwin** [aws/codewhisperer-command-line-discussions#100] [Support ticket]
### Checks

- [X] I ran `cw doctor` in the affected command line session and it didn't resolve my issue
- [X] I ran `cw restart` and it didn't resolve my issue
- [X] I searched [the discussions in this repo](https://github.com/aws/codewhisperer-command-line-discussions/discussions?q=) and there are no duplicates of my issue

### Expected behaviour

when i'm still using the fig version when I type `gc` it will autocomplete git branches because `gc` in my `~/.zshrc` is an alias of git checkout

### Actual behaviour

nothing happens

**Comment by grant0417** [comment]
This is discussed in #21, a fix will be released soon.
